### ISSUE TITLE
APNG compression

### ISSUE THREAD

**Issue by neheb** [subzey/zopfli-png#4]
This tool works great with APNG files. A small question though. The normal behavior of zopfli-png is to not overwrite the file if the IDAT size is larger. However with APNG files I have noticed that the opposite seems to hold.

pngout supports regular PNG and not APNG and as such cannot touch the fdAT chunks. zopfli-png does and works great. However I am noticing that on the first IDAT chunk, zopfli-png seems to overwrite it even if it's smaller. I verify this by running 'pngout /kp /kacTL,fcTL,fdAT file.png' and find that I can get a few extra bytes out of some files. I run pngout before and after compressing with zopfli-png.

tl;dr zopfli-png seems to write the IDAT chunk even if it is bigger on APNG files.
